pom.xml

```diff
@@ -51,4 +60,5 @@
       <url>http://www.kauailabs.com/maven2</url>
     </repository>
   </repositories>
+
```

Comment:
nit. extra newline

Reply:
I tried to fix it, but it didn't take my change, idk.